nix/fluent-bit/default.nix

```diff
@@ -117,6 +117,7 @@ stdenv.mkDerivation rec {
       "-DFLB_RELEASE=ON"
       "-DFLB_DEBUG=OFF"
       "-DFLB_PREFER_SYSTEM_LIBS=Yes"
+      "-DFLB_CORO_STACK_SIZE=24576"
```

Comment:
Do we need to set this for the bundled submodule build as well?

Reply:
It's not strictly necessary when building with glibc, so I guess not.